City_CRUD_API/models.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class City(Base):
+    __tablename__ = "city"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255), nullable=False, unique=True)
```

Comment:
The `name` column is set to be unique and not nullable, which is good for ensuring data integrity. Ensure that any input to this field is validated to prevent errors during database operations.